ariadne/cardano/src/Ariadne/Wallet/Cardano/WalletLayer/Kernel.hs

```diff
@@ -110,12 +113,14 @@ passiveWalletLayerCustomDBComponent logFunction keystore acidDB = do
 
             , pwlApplyBlocks           = liftIO . invoke . Actions.ApplyBlocks
             , pwlRollbackBlocks        = liftIO . invoke . Actions.RollbackBlocks
+            , pwlGetPassiveWallet      = wallet
```

Comment:
This should be _definitely_ not exposed. It can be replaced with `getWalletDB` though, or, better yet, `pwlCleanupDBPeriodically`.